### PR TITLE
Dev/leo

### PR DESCRIPTION
**Pull request by Ugohchukwuma1234**

### CODE REVIEW

File: README.md
Comment (onegeniuslykdat) [inline README.md:1]: Oh so we are also uploading a2.html to this repo

Comment (onegeniuslykdat) [inline README.md:1]: What's the presentation part? I am not clear on it.

**Comment by Ugohchukwuma1234** [comment]
I dont know bro… others did it

File: README.md
Comment (Victorbyke) [reply README.md:1]: I thought it was to be submitted at learn.jcu

Comment (Ugohchukwuma1234) [reply README.md:1]: To be honest, na so I see am for the group wey we use as reference o

Comment (Ugohchukwuma1234) [reply README.md:1]: Our video presentation, seems like they uploaded it on YouTube

**Review by onegeniuslykdat — APPROVED** [review]

**Pull request merged**